cinema/views.py

```diff
@@ -127,6 +127,28 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "genres",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by genres (ex. ?genres=1)",
+            ),
+            OpenApiParameter(
+                "title",
+                type={"type": "list", "items": {"type": "str"}},
```

Comment:
The OpenAPI parameter 'title' is defined with a type of list, but it should be a string since the filter is applied to a single title string. Update the type to `{'type': 'string'}`.